content/example_cf_templates.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Example of using CF Templates"
+---
+
+
+#### Click below to add a CloudFormation Stack
```

Comment:
Does this mean, contributors can click on the template to use it or should you change this to --> "#### Use the template below for CloudFormation Stack"

Reply:
It's click to use